contracts/neutron_interchain_txs/src/contract.rs

```diff
@@ -639,3 +644,19 @@ pub fn reply(deps: DepsMut, env: Env, msg: Reply) -> StdResult<Response> {
         ))),
     }
 }
+
+fn min_ntrn_ibc_fee(fee: IbcFee) -> IbcFee {
+    IbcFee {
+        recv_fee: fee.recv_fee,
```

Comment:
why don't you filter recv_fee?

Reply:
I cannot request changes since I own this PR :cry:

Reply:
It's always 0 so no need to filter?